DEMO_GUIDE.md

```
@@ -0,0 +1,385 @@
+
+## 📝 Quick Reference
+
+### Important URLs
+- **Dashboard**: http://localhost:3000
+- **Simulator**: http://localhost:3000/simulator
+- **Policies**: http://localhost:3000/policies
+- **Backend API**: http://localhost:3002
+- **Health Check**: http://localhost:3002/health
+
+### API Keys for Testing
+| Tier | API Key | Rate Limit |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)